wasmtime/context.go

```diff
@@ -82,12 +83,14 @@ func (c *Context) memory() ([]byte, error) {
 }
 
 func (c *Context) AllocMemory(size int32) (int32, error) {
-	memoryAllocator := c.Instance.GetFunc(c.Store, "allocate")
-	if memoryAllocator == nil {
-		return 0, errors.New("function 'allocate' does not exist")
+	if c.allocator == nil {
```

Comment:
When using the Wasmtime pool, instances will undergo reuse. Will the allocator expire?